### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by Camilovelag**
# Good day! 🌤️

This pull request is intended to connect the app to an existing API to create and remove books in a remote server.

## 🛠️ Changes made

✔️ Fetch book list from Bookstore API on Books page load.
✔️ Refactor _add book_ and _remove book_ features to persist changes in the server.

I'm expecting feedback.

Happy coding!  👏👏👏

### CODE REVIEW

File: src/api/bookstoreApi.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+const appId = 'A5M1SMeOFloqjitxesLY';
+
+const createApp = async () => {
+  const url = `${api}/apps/`;
+  const response = await fetch(url, {
+    method: 'POST',
+  });
+  const data = await response.text();
+  // Needed number to create app
+  console.log(data); // eslint-disable-line
```
Comment (CollinsTatang) [inline src/api/bookstoreApi.js]: - [x] Please remove all  `console.log()`  from your codebase it exposes important data to the user browser console

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Camilovelag , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/api/bookstoreApi.js (hunk 1 above)
Comment (Camilovelag) [reply src/api/bookstoreApi.js]: Done! 😃

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Camilo, 
## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**